packages/lesswrong/lib/collections/collections/helpers.js

```diff
@@ -0,0 +1,19 @@
+import Collections from './collection.js';
+import Books from '../books/collection.js';
+import Sequences from '../sequences/collection.js';
+
+Collections.getAllPostIDs = async (collectionID) => {
```

Comment:
This does not account for posts in Books. We should fix that, or make sure we don't have any posts in Books anymore.

Reply:
Yep, this is a bug. Will address before merging.